src/cli/Generate.cpp

```diff
@@ -0,0 +1,110 @@
+/*
+ *  Copyright (C) 2018 KeePassXC Team <team@keepassxc.org>
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 2 or (at your option)
+ *  version 3 of the License.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <cstdlib>
+#include <stdio.h>
+
+#include "Generate.h"
+
+#include <QCommandLineParser>
+#include <QTextStream>
+
+#include "core/PasswordGenerator.h"
+
+Generate::Generate()
+{
+    this->name = QString("generate");
+    this->description = QObject::tr("Generate a new random password.");
```

Comment:
Remove explicit this

Reply:
Because this is like all other cli Command do it, do I have to change them all?

Reply:
Also fixed this for every cli command 🎉